### PR TITLE
Enable custom styles

### PR DESCRIPTION
**Pull request by igorp1**
__what__

This PR adds the interface to allow integrators to customise the styles for the duffel card component. 
This PR is counterpart to https://github.com/duffelhq/token-proxy/pull/260.

### CODE REVIEW

File: src/components/DuffelCardForm/DuffelCardForm.tsx [hunk 1]
```diff
@@ -75,6 +56,26 @@ export const DuffelCardForm: React.FC<DuffelCardFormProps> = ({
     );
   }
 
+  function postMessageWithStyles() {
+    if (!iFrameReference.current) {
+      throw new Error(
+        "Attempted to call `sendMessageToStoreCardForTemporaryUse` with empty iFrameReference"
```
Comment (andrejak) [inline src/components/DuffelCardForm/DuffelCardForm.tsx:62]:
```suggestion
        "Attempted to call `postMessageWithStyles` with empty iFrameReference"
```

File: src/components/DuffelCardForm/DuffelCardForm.tsx [hunk 2]
```diff
@@ -75,6 +56,26 @@ export const DuffelCardForm: React.FC<DuffelCardFormProps> = ({
...
+      );
+    }
+
+    const iFrame = iFrameReference.current;
+    if (!iFrame.contentWindow) {
+      throw new Error(
+        "Attempted to call `sendMessageToStoreCardForTemporaryUse` but the iFrame contentWindow is null"
```
Comment (andrejak) [inline src/components/DuffelCardForm/DuffelCardForm.tsx:69]:
```suggestion
        "Attempted to call `postMessageWithStyles` but the iFrame contentWindow is null"
```

**Review by andrejak — APPROVED** [review]

File: src/components/DuffelCardForm/DuffelCardForm.tsx (hunk 1 above)
Comment (igorp1) [reply src/components/DuffelCardForm/DuffelCardForm.tsx]: good catch! Fixed it!

File: src/components/DuffelCardForm/DuffelCardForm.tsx (hunk 2 above)
Comment (igorp1) [reply src/components/DuffelCardForm/DuffelCardForm.tsx]: Fixed here too!

**Pull request merged**